### ISSUE TITLE
Get large and small profile picture in one go 

### ISSUE THREAD

**Issue by rajeevriitm** [arsduo/koala#557]
Currently I can get only one type of picture in one go. When I include both ``small`` and ``large`` type in ``picture``  type , I get an error saying ``Koala``  support a field ( ``picture`` ) only once in the request. Is there any way to do it without using another request to get picture  ?

**Comment by rostixman** [comment]
I have same problem...

**Comment by arsduo** [comment]
Hmm. Is that supported by the Facebook API?

If you're able to test this in the Facebook Graph API explorer (https://developers.facebook.com/tools/explorer/), that would be hugely useful. If it's not possible we can close this, if it is but Koala doesn't support it, I'll get that fixed.

Thanks!

**Comment by arsduo** [comment]
Since it's been three weeks I'm going to close this, but if you are still having the issue and can see it working in the graph explorer, definitely reopen this or create a new issue with that info. Thanks!

**Comment by rajeevriitm** [comment]
Sorry for the late response. I checked Facebook API and its not possible to do it there. So I guess two requests are necessary . Thanks for the response.

**Comment by arsduo** [comment]
Sorry to hear that :/ One way to reduce the outgoing requests would be to use a batch API call!
